### PR TITLE
Enhancement in tsuru unit names (#1160)

### PR DESCRIPTION
**Pull request by lucasweiblen**
Closes issue #1160.

### CODE REVIEW

File: provision/docker/actions_test.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func (s *S) TestInsertEmptyContainerInDBForward(c *check.C) {
 	c.Assert(cont.AppName, check.Equals, app.GetName())
 	c.Assert(cont.Type, check.Equals, app.GetPlatform())
 	c.Assert(cont.Name, check.Not(check.Equals), "")
-	c.Assert(cont.Name, check.HasLen, 20)
+	c.Assert(cont.Name, check.HasLen, 26)
```
Comment (fsouza) [inline provision/docker/actions_test.go]: Can you also check that it begins with "appname/"?

Something like:

```
c.Assert(strings.HasPreffix(cont.Name, app.GetName() + "/"), check.Equals, true)
```

Comment (lucasweiblen) [inline provision/docker/actions_test.go]: sure

**Comment by lucasweiblen** [comment]
are you guys having trouble with Travis? any ideas why this build is breaking?

**Comment by fsouza** [comment]
Yep, something is wrong with godep at Go tip. Nothing related to your changes, I will try to send a bug report to godep tomorrow.

**Comment by fsouza** [comment]
Thanks for contributing, again! :-)

**Comment by fsouza** [comment]
Argh, actually I forgot to check if Docker accepts slashes in the name of the container, and turns out it doesn't. I will use a dash instead.

**Comment by lucasweiblen** [comment]
hehehe it happens! :smile:

**Pull request merged**